### ISSUE TITLE
Inference results on VoxConverse differ from provided output

### ISSUE THREAD

**Issue by thanhnam001** [BUTSpeechFIT/DiaPer#14]
Hi, I posted a comment in other post but didn't receive any response, so I'm opening a new issue here.
I'm currently running inference with your pretrained model on the VoxConverse dataset. I used the [infer_16k_10attractors.yaml](https://github.com/BUTSpeechFIT/DiaPer/blob/main/examples/infer_16k_10attractors.yaml) config but cannot get the same output as yours. I've set up some code here on [kaggle](https://www.kaggle.com/code/namvothanh/testdiaper) to show the difference. 
I also ran some files in RAMC dataset and the outputs are the same as your published results. Did I miss any step?
Thanks.

**Comment by fnlandini** [comment]
Hi, sorry for the delay.
It is strange if you managed to replicate the results with another model but not with one in particular. I am not sure but one possibility is that the audios for VoxConverse have changed since we downloaded them originally in 2020 and that leads to different results. If you evaluate the whole set, do you obtain similar results?

**Comment by thanhnam001** [comment]
Hi, sorry for the late reply.
I ran inference on the whole test set of VoxConverse and RAMC, as you suggested. The outputs of RAMC are identical to your published result, while the outputs of VoxConverse show the opposite trend. (I did not run `dscore` evaluation, I only compared its content)
I used the audio source for VoxConverse from [github](https://github.com/joonson/voxconverse). In the README file, I did not see any changes in audio files; they only changed the ground truth files because some of those were not precise.
If you still keep the original data that you downloaded in 2020, can you rerun the evaluation to double-check?
Also, I've attached the outputs of RAMC and VoxConverse here; perhaps they will help you diagnose the problem.
Thanks.

[output_pretrainedSC_noFT_voxconverse_test.zip](https://github.com/user-attachments/files/21029397/output_pretrainedSC_noFT_voxconverse_test.zip)
[output_pretrainedSC_noFT_ramc_test.zip](https://github.com/user-attachments/files/21029398/output_pretrainedSC_noFT_ramc_test.zip)

**Comment by Cerbera-odollam** [comment]
Hi @thanhnam001,
I had the same problem, but creating a new conda environment solved it for me.
I think the issue might be related to the transformers version.

When I used transformers version 4.39.0, I got a DER of 57.8 on the MSDwild dataset using the dscore GitHub repo.
However, with transformers version 4.30.0.dev0, the DER dropped to 15.22.

The performance may not fully match the results reported in the paper due to some minor differences, but I believe the major issue is the PyTorch version. So I recommend checking that first.

Let me know if you discover anything else.
